src/utils/use-product-handler.js

```diff
@@ -0,0 +1,239 @@
+import { useState, useMemo, useCallback } from 'react';
```

Comment:
Please, rename it to `use-product-handlers`

Reply:
renamed